tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_beacon.c

```diff
@@ -229,6 +235,128 @@ static void test_rx_on_key_refresh(void)
 	PASS();
 }
 
+static void test_tx_beacon_init(void)
```

Comment:
No necessity in `test_tx_beacon_init` and `test_rx_beacon_init`. Just reuse already existing `test_tx_init` and `test_rx_init`. Incoming waiting time in `bt_mesh_test_cfg_set` is not related to beacon timing.